airflow/operators/bash_operator.py

```diff
@@ -35,7 +35,6 @@
 class BashOperator(BaseOperator):
     """
     Execute a Bash script, command or set of commands.
-
```

Comment:
Deleting this line will fail the Sphinx rendering.

Reply:
@XD-DENG I added a fix. Tests pass